tests/executor_test.py

```diff
@@ -65,6 +65,10 @@ def test_executor_dependent(backendopt):
 
 def test_executor_debug_symmetry(backendopt):
     for datatype in backendopt:
+        if datatype == "taco":
+            # Taco addition (line 76) will output sparse matrix even though the input is dense.
+            # This will make the format check fail.
```

Comment:
Does this sound like a bug to TACO? Maybe followup with them.

Reply:
Yes, just opened an issue in taco.